scripts/CEE/hs-must-gather/script.sh

```diff
@@ -34,8 +36,16 @@ collect_must_gather() {
   # Fetch the HostedCluster name
   hc_name=$(oc get HostedCluster.hypershift.openshift.io -l api.openshift.com/id="${CLUSTER_ID}" -n "${hcp_ns}" -o json | jq -r '.items[0].metadata.name')
 
-  echo "Executing hypershift dump on ${hcp_ns} for cluster ${hc_name}"
-  hypershift dump cluster --dump-guest-cluster --namespace "${hcp_ns}" --name "${hc_name}" --artifact-dir "${DUMP_DIR}" --archive-dump=false
+  HC_DUMP_DIR="hostedcluster-${hc_name}"
+  CP_DUMP_DIR="controlplane-${hc_name}"
+
+  if [ "${DUMP_GUEST_CLUSTER}" = "true" ]; then
```

Comment:
Is there a way we valid the value here is boolean? If user passes other string here, the script would be failed?